doc/nrf/libraries/others/emds.rst

```diff
@@ -128,9 +128,9 @@ The :c:func:`emds_store_time_get` function estimates the required worst-case tim
    t_\text{store} = t_\text{base} + \sum_{i = 1}^n \left(t_\text{entry} + t_\text{word}\left(\left\lceil\frac{s_\text{ate}}{s_\text{block}}\right\rceil + \left\lceil\frac{s_i}{s_\text{block}}\right\rceil \right)\right)
 
 where :math:`t_\text{base}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_BASE_OVERHEAD_US`, :math:`t_\text{entry}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_ENTRY_OVERHEAD_US` and :math:`t_\text{word}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_WRITE_ONE_WORD_US`.
-:math:`s_i` is the size of the :math:`i`\ th entry in bytes and :math:`s_\text{block}` is the number of bytes in one word of flash.
+:math:`s_i` is the size of the :math:`i`\ th entry in bytes and :math:`s_\text{block}` is the number of bytes in one word of flash, while for RRAM the write-buffer size ``i`` number of 128-bit words (16 bytes).
```

Comment:
This sentence doesn't make sense when I try to read it? Is $s_i =$ the number of 128-bit words used by the entry? Or is $s_\text{block} =$ 16? It would be good if you used the terms from the formula (`` :math:`s_i\` `` and `` :math:`s_\text{block}` ``) and explain what values they should have in the RRAM case (what is "write-buffer size" in this case for instance? I can't tell where to put 128 and/or 16 into the formula)